packages/edition-twig/patternlab-config.json

```diff
@@ -0,0 +1,144 @@
+{
+  "engines": {
```

Comment:
I love the potential for this key! cc @geoffp

Reply:
Thanks! I think so too. I initially had `twigConfig`, but then realized that `engines.twig` made way more sense.